Classes/common/HTTP/CDTURLSession.m

```diff
@@ -40,6 +42,7 @@ - (instancetype)initWithDelegate:(id<NSURLSessionDelegate>)delegate callbackThre
         _thread = thread;
         
         NSURLSessionConfiguration *config = [NSURLSessionConfiguration defaultSessionConfiguration];
+        _requestInterceptors = @[];
```

Comment:
Right, it should be immutable, classes shouldn't be able to modify the interceptors on the request. Like you can't in Java.